src/wasm/wasm-emscripten.cpp

```diff
@@ -718,6 +718,11 @@ void AsmConstWalker::visitCall(Call* curr) {
         if (set) {
           assert(set->index == get->index);
           arg = set->value;
+        } else {
+          Fatal() << "local.get of unknown in arg0 of call to " << import->base
```

Comment:
I might no be clear from the function name what the problem is.  Perhaps mention that issue is related to EM_JS macro?

Reply:
Done.